shared/utils/enable-derivatives.py

```diff
@@ -261,16 +264,18 @@ def scrape_benchmarks(root_element, namespace, dest):
     for namespace, benchmark in benchmarks:
         if not add_derivative_cpes(benchmark, namespace, mapping):
             raise RuntimeError(
-                "Could not add derivative OS CPEs to Benchmark '%s'." % (benchmark)
```

Comment:
I suggest raising the line length limit from 80 to 100. Although majority of lines can and should be under 80 chars long, I think that enforcing this rule upon every line is more like self-torture. PEP8 actually states that it is OK to increase to width of 99: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
sure, I'd be all for that but it's unrelated to this PR. Let's make a separate PR that changes the pep8 settings.

Reply:
I agree, fair enough.